src/specs/index.js

```diff
@@ -6,6 +6,7 @@ describe('ui-harness', function() {
   require('./it-server.spec');
   require('./PropTypes.spec');
   require('./page.spec');
+  require('./typescript.spec.tsx')
```

Comment:
I wonder if you'd get automatic file lookup resolution on the `.tsx` extension.  That would be nice.  Like the way we don't have to write `.js` on each file.